versioned/mongodb/src/main/java/com/dremio/nessie/versioned/store/mongodb/MongoStoreConfig.java

```diff
@@ -28,8 +28,38 @@ public String getDatabaseName() {
   }
 
   @Default
-  public String getTablePrefix() {
-    return "";
+  public String getRefTableName() {
```

Comment:
I think it's a better to have the strings as constants up in the class, outside of any function, and just return the constants in the functions.